controllers/configmap_controller.go

```diff
@@ -178,7 +201,7 @@ func (r *ConfigMapReconciler) reconcileNodes(ctx context.Context, windowsInstanc
 	// Get the list of instances that are expected to be Nodes
 	instances, err := r.parseInstances(windowsInstances.Data, nodes)
 	if err != nil {
-		return errors.Wrap(err, "unable to parse hosts from ConfigMap")
+		return errors.Wrap(err, "unable to parse instances from ConfigMap")
```

Comment:
This change should go into `[wmco] Refactor BYOH instance retrieval helper` because changing it there is warranted but not in this commit.